modules/questions.py

```diff
@@ -564,7 +563,7 @@ def code_info(self, *, with_num: bool = False) -> str:
         """Print info about query in code block"""
         d = {"status": self.status, "tag": self.tag}
         if with_num:
-            d["max_num_of_questions"] = self.max_num_of_questions
+            d["max_num_of_questions"] = self.max_num_of_questions #type:ignore
```

Comment:
any plans how to make the type correct in the future?

Reply:
initially I annotated it as `dict[str, Union[str, int, None]]` but it's excessively wordy for a small thing like this IMO
alternatively, I can just make it `dict[str, Any]`
(also, I don't know how many people other than me use type checking in this project, so maybe on particularly annoying lines it's better if I just silence it than annotate)